mizar/common/common.py

```diff
@@ -444,6 +475,13 @@ def get_default_itf():
     default_itf="eth0"
     ret, data = run_cmd("ip route")
     data = [i for i in data.split('\n') if i]
+
+    if os.getenv('FEATUREGATE_OFFLOAD_XDP', 'false').lower() in ('true', '1'):
+        for offload_xdp_itf in support_offload_xdp_itf_names():
```

Comment:
the intent of this function is to return the default route interface name (which may not necessarily be eth0). I think this changes that basic assumption. Do the unit tests pass with featuregate enabled?

Reply:
XDP programs are loaded at the default interface. With FEATUREGATE_OFFLOAD_XDP on, I think the default interface should be the one that supports XDP_OFFLOAD mode. If all interfaces can not support offloading, this function still works as before even with FEATUREGATE_OFFLOAD_XDP on.

Reply:
Got it. I have removed my codes here.